### DISCUSSION TITLE
Mode packages?

### DISCUSSION THREAD

**Discussion by Anixx** [ramensoftware/windhawk#59] [General]
Is there a way to include into one mod several .md.cpp files which are united by the same topic but should be injected into different processes?

For instance, I want to set certain attribute for most processes to value A and to a few exceptions to value B. It would not be optimal, I think, to make two separate mods. Or there is another way to do so?

**Comment by m417z** [comment]
> Is there a way to include into one mod several .md.cpp files which are united by the same topic but should be injected into different processes?

No, currently that's not possible.

> For instance, I want to set certain attribute for most processes to value A and to a few exceptions to value B. It would not be optimal, I think, to make two separate mods. Or there is another way to do so?

You can have a mod that injects into all the processes you need, and checks in which process it runs to run the desired logic. You can look at "Text Replace" and "Timer Resolution Control" which check in which process they run.
